### ISSUE TITLE
ARM Chromebook Graphics Problems

### ISSUE THREAD

**Issue by JVGA8837** [LAX1DUDE/eaglercraft#43]
![image_2022-01-30_202018](https://user-images.githubusercontent.com/67811123/151726773-62a9cc0e-9533-4090-bf91-a0684700689a.png)
Game runs at a stable 10 - 20 fps but graphics are all screwed up

**Comment by MapleTheCoder** [comment]
Open developer tools (Click the three dots at the top right hand corner of your chrome window and navigate to -> More Tools -> Developer Tools) and then hard reload the page and empty cache (right click the reload page button at the top left hand corner of the screen and click hard reload and empty cache) and then navigate to the console tab in the developer tools. Post the console's content when the page has loaded all the way to the screen shown in the screenshot.

**Comment by sussygamedeveloper** [comment]
That might not be possible because chances is he is probably on a school Chromebook. I also have 2 friends who have ARM Chromebook and the graphics are fucked up.

**Comment by JVGA8837** [comment]
Yes, it is a school one but unfortunately dev console is disabled

**Comment by MapleTheCoder** [comment]
I guess we will have to wait till someone has a ARM chromebook with dev tools enabled? 

> That might not be possible because chances is he is probably on a school Chromebook. I also have 2 friends who have ARM Chromebook and the graphics are fucked up.

Maybe one of your friends can see if they can use dev console to help us out?

**Comment by JVGA8837** [comment]
![IMG_0895](https://user-images.githubusercontent.com/67811123/151731474-2319285a-32c5-48c9-8540-d9091698b9f5.jpg)
Forgot I had a personal Chromebook its not arm its an AMD stoney ridge one but the graphics are fine on here

**Comment by sussygamedeveloper** [comment]
I will try to run chrome os in a arm qemu vm and send the results.

**Comment by sussygamedeveloper** [comment]
that's smart i have one at home but its a rpi 3, i can't test rn cause im at school and didn't set up remote desktop for my rpi.

**Comment by JVGA8837** [comment]
I considered that but i don't have any arm based laptops other than this crap school one and my rpi 4

**Comment by sussygamedeveloper** [comment]
ok

**Comment by JVGA8837** [comment]
Testing Manjaro ARM currently will update soon.

**Comment by sussygamedeveloper** [comment]
Nice!

**Comment by JVGA8837** [comment]
Unfortunately, I switched to raspbian since manjaro stopped booting for some reason. But on raspbian graphics display fine, all though I will test using an rpi 3

**Comment by JVGA8837** [comment]
![2022-02-01-004607_1920x1080_scrot](https://user-images.githubusercontent.com/67811123/151896838-15906914-3c3c-4409-8b1a-ad29541590ff.png)
rpi 3 throws a webgl error, tried online and offline with no difference.

**Comment by sussygamedeveloper** [comment]
yea happened with me on my rpi3. its probably a driver issue.

**Comment by sussygamedeveloper** [comment]
@JVGA8837 does the loading screen look like this https://github.com/LAX1DUDE/eaglercraft/issues/58

**Comment by JVGA8837** [comment]
yes exactly like that, but his game seems to have solid color graphics and not glitched colors like arm ones do.

**Comment by JVGA8837** [comment]
I’ve forked this and I’ll see if I can recompile the gl libraries to make a native arm port since everything else works as intended

**Comment by JVGA8837** [comment]
Well I just found out that teavm doesn’t have any arm libraries. And I don’t think with my knowledge of Java will be able to pull this off. I’ll try but I probably won’t get far

**Comment by GojoSatoruSealed** [comment]
![Screenshot 2022-02-10 10 47 55](https://user-images.githubusercontent.com/81868696/153381354-08297771-e66f-4dc0-b553-c1cb4b6f6c50.png)
@MapleTheCoder
